packages/perseus-editor/src/components/__stories__/locked-point-settings.stories.tsx

```diff
@@ -43,18 +43,25 @@ export const Controlled: StoryComponentType = {
     },
 };
 
+Controlled.parameters = {
+    chromatic: {
+        // Disabling because this doesn't test anything visual, just behavior.
+        disableSnapshot: true,
+    },
+};
+
 /**
  * In some cases, the locked point may be shown or hidden from the graph,
  * such as when the point is used to define a locked line.
  *
- * Use `onToggle` and `toggled` to update the visibility of the locked point.
- * When `onToggled` is passed in, a switch will be rendered to allow
+ * Use `onExtrasToggle` and `extrasToggled` to update the visibility of the locked point.
+ * When `onextrasToggled` is passed in, a switch will be rendered to allow
```

Comment:
Typo: Should this be "When `onExtrasToggled` is passed in..."?

Reply:
Yes! But this is also going away in my next PR, so I'm going to leave it as is for now